packages/gatsby-recipes/src/renderer/render.js

```diff
@@ -223,18 +229,16 @@ const render = (recipe, cb, inputs = {}, isApply, isStream, name) => {
   const emitter = mitt()
   const plan = {}
 
-  const queue = new Queue(
-    async (job, cb) => {
-      const result = await job
-      cb(null, result)
-    },
-    { concurrent: 5 }
```

Comment:
Would removing this (the concurrency) not have a detrimental effect on runtime perf when fetching resources async? Or is it basically "infinite" now?

Reply:
my thinking here is that concurrency isn't a global concern but a concern of the resource code itself — it just isn't knowable what the right concurrency is at the global level. The previous setting of 5 was almost certainly slowing things down unreasonably.

We _will_ see problems at the resource level because of this change but we can develop techniques/libraries to help each resource pick the right level of concurrency.